src/distributions/mod.rs

```diff
@@ -328,6 +328,7 @@ pub struct Standard;
 
 #[cfg(all(test, feature = "std"))]
 mod tests {
+    use std::{vec::Vec, println};
```

Comment:
I think we can simply remove the `println` usage and replace `collect` with an explicit loop over `0..100`,

Reply:
I'll replace it with a sum and enable the tests on `no_std`.